lib/seo_meta.rb

```diff
@@ -34,8 +34,13 @@ def is_seo_meta(options = {})
     end
 
     # Let SeoMetum know about the base
-    ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
-                    :class_name => self.name
+    if ActiveRecord::VERSION::STRING >= '5.0.0'
+      ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
+                      class_name: self.name, optional: true
```

Comment:
We could change [the gemspec](https://github.com/parndt/seo_meta/blob/d9414fc63546d089036496342d343d5eb0bbc6d1/seo_meta.gemspec#L21) to require version 4 and above?